components/Select/index.scss

```diff
@@ -23,21 +23,23 @@
   * {
     box-sizing: border-box;
   }
+}
+
+.caret {
+  position: absolute;
+  right: 15px;
+  top: ($input-height-base / 2.5);
+  font-size: 10px;
+  z-index: 3;
+  pointer-events: none;
 
-  &:after {
-    content: url('data:image/svg+xml; utf8, %3Csvg%20version%3D%221.1%22%20%20xmlns%3D%22http%3A%2F%2Fwww.w3.org%2F2000%2Fsvg%22%20xmlns%3Axlink%3D%22http%3A%2F%2Fwww.w3.org%2F1999%2Fxlink%22%20x%3D%220px%22%20y%3D%220px%22%0A%09%20viewBox%3D%220%200%2013.4%209.1%22%20style%3D%22enable-background%3Anew%200%200%2013.4%209.1%3B%22%20xml%3Aspace%3D%22preserve%22%3E%0A%3Cg%3E%0A%09%3Cpath%20d%3D%22M13%2C2.8L7.1%2C8.6C7%2C8.7%2C6.9%2C8.8%2C6.8%2C8.8c-0.1%2C0-0.3-0.1-0.4-0.2L0.5%2C2.8C0.4%2C2.6%2C0.4%2C2.5%2C0.4%2C2.4c0-0.1%2C0-0.3%2C0.2-0.4%0A%09%09l1.3-1.3c0.1-0.1%2C0.2-0.2%2C0.4-0.2s0.3%2C0.1%2C0.4%2C0.2l4.2%2C4.2L11%2C0.7c0.1-0.1%2C0.2-0.2%2C0.4-0.2c0.1%2C0%2C0.3%2C0.1%2C0.4%2C0.2L13%2C2%0A%09%09c0.1%2C0.1%2C0.2%2C0.2%2C0.2%2C0.4C13.2%2C2.5%2C13.1%2C2.6%2C13%2C2.8z%22%2F%3E%0A%3C%2Fg%3E%0A%3C%2Fsvg%3E');
+  svg {
     width: 10px;
     height: 10px;
-    position: absolute;
-    right: 15px;
-    top: ($input-height-base / 3);
-    font-size: 10px;
-    z-index: 3;
-    pointer-events: none;
+  }
 
-    html.ie10 & {
-      z-index: 0;
-    }
+  html.ie10 & {
+    z-index: 0;
```

Comment:
I know this isn't new, but could you explain why IE 10 has problems with z-index: 3? Looks a little bit weird from a css perspective. Also I'm not a fan of this reverse nesting in sass.

Reply:
I believe this explains it https://stackoverflow.com/questions/23344655/ie-10-not-respecting-z-index :D Although I am not exactly sure. This has been a hack we've been using since the beginning.